modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java

```diff
@@ -1127,7 +1127,7 @@ private void _addSystemObjectFields(
 		_objectFieldLocalService.addSystemObjectField(
 			userId, objectDefinition.getObjectDefinitionId(),
 			ObjectFieldConstants.BUSINESS_TYPE_LONG_INTEGER, dbColumnName,
-			dbTableName, ObjectFieldConstants.DB_TYPE_LONG, true, true, null,
+			dbTableName, ObjectFieldConstants.DB_TYPE_LONG, false, false, null,
```

Comment:
@gustavolimav I believe they should continue being marked as true here, because they are actually indexable. Not by us (Objects code), but they are.

Reply:
@gabrielwas technically it should be false (which is why my suggestion was to maybe hide the "searchable" panel), but it doesnt really have an effect either way, so could be true. the only effect will be on the UI, so if you want all switches to be "on" but deactivated, we could set to true. but i think they should all be true or all be false

Reply:
maybe leave false for now, and once you decide on what you want for the UI you can switch them all to true?

Reply:
I'm considering that we may confuse the user with these set as false, but it's actually indexed. 
Agree with you that the approach should be all switches to be "on" but deactivated. And also, to make this, we should perform an upgrade process for the already created ones. 
Can we wait align this with the BPM Team Product Manager? (just for this commit)

Reply:
im not a UX expert but just want to point out that it also seems like it could be confusing to show a deactivated switch for something that can never be activated. if you just hide the panel, no need to worry about the values, no need for upgrade process etc. but ya the BPM Team Product Manager can decide.
also, the current behavior is that all other switches are currently off, ID is the only one that is on, so this change is less intrusive to the current behavior